docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -6,141 +6,197 @@
 
-----
-oc adm policy add-cluster-role-to-user collect-audit-logs -z logcollector
-----
+CPU and memory limits::
+As with all cluster resources, use the values shown above as a reference point and adjust as necessary. If your pipeline is complex, you may need more collector resources; if you have a more straightforward pipeline, you may need less.
+* `resources.limits` describes the maximum amount of compute resources allowed
 
-==== Management, Resource Allocation & Workload Scheduling
-Configuration of the management state (i.e. managed, unmanaged), resource request and limits, tolerations, and node selection
-are part of the new ClusterLogForwarder API.
+* `resources.requests` describes the minimum amount of compute resources required.  Defaults to values of *limits* if not specified
 
-.Previously part of the resource *ClusterLogging* (v5.9)
-[source, yaml]
-----
-apiVersion: logging.openshift.io/v1
-kind: ClusterLogging
-metadata:
-  name: instance
-spec:
-  managementState: Managed
-  collection:
-    resources:
-      limits: {}
-      requests: {}
-    nodeSelector: {}
-    tolerations: {}
-----
-.Now part of v6.0 *ClusterLogForwarder* along with inputs, outputs, filters and pipelines
+
+==== Permissions (Service Accounts)
+The *ClusterLogForwarder* now requires a cluster administrator to provide a service account, with correct RBAC permissions.  This service account is now a required part of the configuration.
+
+.Required Service Account
 [source, yaml]
 ----
 apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance
+  name: my-forwarder
 spec:
-  managementState: Managed
-  collector:
-    resources:
-      limits: {}
-      requests: {}
-    nodeSelector: {}
-    tolerations: {}
+  serviceAccount:
+    name: logging-admin
 ...
 ----
+Administrators are required to *explicitly grant log collection permissions* to the service account referenced in the *ClusterLogForwarder*.   There are 3 cluster roles that can be bound to: *collect-application-logs*, *collect-infrastructure-logs* and *collect-audit-logs*.
 
-==== Inputs Specifications
+.ClusterRoleBinding for *logging-admin*
+----
+oc adm policy add-cluster-role-to-user collect-application-logs -z logging-admin
+oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z logging-admin
+----
+
+Additionally, if collecting audit logs:
+----
+oc adm policy add-cluster-role-to-user collect-audit-logs -z logging-admin
+----
 
+.Legacy *openshift-logging*
+If your previous forwarder is deployed in the namespace *openshift-logging* and named *instance*, then you've likely been using the service account *logcollector* created by earlier versions of the operator.  You can optionally grant the new RBAC permissions to this SA.
+
+IMPORTANT: To continue using the *logcollector* service account, you still *MUST explicitly grant log collection permissions* by creating a *ClusterRoleBinding* to the necessary roles.
```

Comment:
I think we need to mention removing the ownerref here

Reply:
I'd rather not in this section, as it's listed under "Permissions (Service Accounts)".      I've covered removing the ownerref in each of the sections on Log Storage and Log Visualization.    That should cover it?